provd/devices/ident.py

```diff
@@ -33,7 +33,7 @@ class RequestType(Enum):
     DHCP = 'dhcp'
 
 
-DHCPRequest = Dict[str, Union[str, Dict[str, str]]]
+DHCPRequest = dict[str, Union[str, dict[str, str]]]
```

Comment:
Why changing this?

Reply:
As of Python 3.9 literals can be used directly. This avoids needing to import the types from `typing`

Reply:
Ah ok, then we have an unused import of `Dict`

Reply:
Oh, good catch. Thanks. I'll fix. Provd doesn't have any linters it seems.